services/ui-src/src/components/CoreQuestionWrapper/index.tsx

```diff
@@ -8,10 +8,12 @@ export interface Props {
 export const CoreQuestionWrapper = ({ label, children }: Props) => {
   return (
     <CUI.Box my={4}>
-      <CUI.FormLabel fontWeight="bold" fontSize="lg" data-cy={label}>
-        {label}
-      </CUI.FormLabel>
-      {children}
+      <CUI.Heading>
+        <CUI.FormLabel fontWeight="bold" fontSize="lg" data-cy={label}>
+          {label}
+        </CUI.FormLabel>
+      </CUI.Heading>
+      <fieldset>{children}</fieldset>
```

Comment:
```suggestion
        <CUI.FormLabel fontWeight="bold" fontSize="lg" data-cy={label} as="h2">
          {label}
        </CUI.FormLabel>
      <fieldset>{children}</fieldset>
```
Not at all necessary, but I think this would also work here.

Reply:
this might actually save me the trouble of having to fix all the tests, I'm going to try this

Reply:
this is cleaner so I'm going with this but unfortunately the tests still broke :(

Reply:
Also I did see what you mean with the empty return to. It was already there and I tried filtering it out and it makes it so that you can't go back to the measures list from a measure so I think it was meant to be there.